tests/face_extend_test.py

```diff
@@ -1,19 +1,141 @@
-"""Tests for features add to honeybee by honeybee_radiance."""
+"""Tests the features that honeybee_radiance adds to honeybee_core Face."""
 from honeybee.face import Face
-from honeybee_radiance.properties import FaceRadianceProperties
+from honeybee.boundarycondition import boundary_conditions
 
+from honeybee_radiance.properties.face import FaceRadianceProperties
+from honeybee_radiance.modifier.material import Plastic
 
-face = Face.from_vertices(
-    'wall_face', [[0, 0, 0], [10, 0, 0], [10, 0, 10], [0, 0, 10]])
+from ladybug_geometry.geometry3d.pointvector import Point3D, Vector3D
+from ladybug_geometry.geometry3d.face import Face3D
+
+import pytest
 
 
 def test_radiance_properties():
+    """Test the existence of the Face radiance properties."""
+    face = Face.from_vertices(
+        'wall_face', [[0, 0, 0], [10, 0, 0], [10, 0, 10], [0, 0, 10]])
     assert hasattr(face.properties, 'radiance')
     assert isinstance(face.properties.radiance, FaceRadianceProperties)
+    assert not face.properties.radiance.is_modifier_set_on_object
+
+
+def test_default_modifiers():
+    """Test the auto-assigning of modifiers by face type and boundary condition."""
+    vertices_wall = [[0, 0, 0], [0, 10, 0], [0, 10, 3], [0, 0, 3]]
+    vertices_floor = [[0, 0, 0], [0, 10, 0], [10, 10, 0], [10, 0, 0]]
+    vertices_roof = [[10, 0, 3], [10, 10, 3], [0, 10, 3], [0, 0, 3]]
+
+    wf = Face.from_vertices('wall', vertices_wall)
+    assert wf.properties.radiance.modifier.name == 'generic_wall_0.50'
```

Comment:
A better test is probably comparing the full modifier which we can import from modifier set. In that case we don't have to update the name if we decide to change the modifier in the modifier-set at some point.

Reply:
Fair point. I'll change it.